app/javascript/packages/eslint-plugin/configs/recommended.js

```diff
@@ -2,72 +2,495 @@ const { isDependency } = require('@aduth/is-dependency');
 
 const config = {
   extends: /** @type {string[]} */ ([]),
-  plugins: /** @type {string[]} */ ([]),
+  plugins: ['import'],
   env: {
     es6: true,
+    node: true,
+  },
+  parserOptions: {
+    ecmaFeatures: {},
+    sourceType: 'module',
+  },
+  settings: {
+    'import/extensions': ['.js', '.jsx'],
+    'import/external-module-folders': ['node_modules'],
+    'import/ignore': ['node_modules'],
+    'import/resolver': {
+      node: {
+        extensions: ['.js', '.jsx'],
+      },
+    },
   },
   rules: {
-    'array-bracket-spacing': 'off',
-    'arrow-parens': 'off',
-    'arrow-spacing': 'off',
-    'block-spacing': 'off',
-    'brace-style': 'off',
-    'class-methods-use-this': 'off',
-    'comma-dangle': 'off',
-    'comma-spacing': 'off',
-    'comma-style': 'off',
-    'computed-property-spacing': 'off',
-    'consistent-return': 'off',
+    'array-callback-return': [
+      'error',
+      {
+        allowImplicit: true,
+        checkForEach: false,
+      },
+    ],
+    'arrow-body-style': [
+      'error',
+      'as-needed',
+      {
+        requireReturnForObjectLiteral: false,
+      },
+    ],
+    'block-scoped-var': 'error',
+    camelcase: [
+      'error',
+      {
+        properties: 'never',
+        ignoreDestructuring: false,
+        ignoreImports: false,
+        ignoreGlobals: false,
+      },
+    ],
+    'constructor-super': 'error',
     curly: ['error', 'all'],
-    'dot-location': 'off',
-    'func-call-spacing': 'off',
-    'func-names': 'off',
-    'function-call-argument-newline': 'off',
-    'function-paren-newline': 'off',
-    'generator-star-spacing': 'off',
-    'prefer-arrow-callback': 'off',
-    'import/prefer-default-export': 'off',
-    'import/extensions': ['off', 'never'],
-    'import/no-extraneous-dependencies': 'error',
-    indent: 'off',
-    'jsx-quotes': 'off',
-    'key-spacing': 'off',
-    'keyword-spacing': 'off',
-    'max-len': 'off',
-    'max-classes-per-file': 'off',
-    'newline-per-chained-call': 'off',
+    'default-case': 'error',
+    'default-case-last': 'error',
+    'default-param-last': 'error',
+    'dot-notation': [
+      'error',
+      {
+        allowKeywords: true,
+        allowPattern: '',
+      },
+    ],
+    'eol-last': ['error', 'always'],
+    eqeqeq: [
+      'error',
+      'always',
+      {
+        null: 'ignore',
+      },
+    ],
+    'for-direction': 'error',
+    'getter-return': [
+      'error',
+      {
+        allowImplicit: true,
+      },
+    ],
+    'global-require': 'error',
+    'grouped-accessor-pairs': 'error',
+    'guard-for-in': 'error',
+    'import/export': 'error',
+    'import/first': 'error',
+    'import/newline-after-import': 'error',
+    'import/no-absolute-path': 'error',
+    'import/no-amd': 'error',
+    'import/no-cycle': [
+      'error',
+      {
+        maxDepth: '∞',
```

Comment:
LOL at this being the unicode infinity instead of like of the JS `Infinity` constant, probably better JSON compat this way?

Reply:
Heh, hadn't noticed that 😅  I'd guess so? Though I could imagine the literal "Infinity" would work just as well too, and not require looking up the unicode character. 🙃